oak_functions_containers_app/build.rs

```diff
@@ -26,6 +26,10 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         ],
         CodegenOptions {
             build_server: true,
+            extern_paths: vec![ExternPath::new(
+                ".oak.functions",
+                "::oak_functions_service::proto::oak::functions",
+            )],
```

Comment:
I didn't realise this is an option, we may be able to clean up some other hacky imports that I suggested in the past (cc @ipetr0v )